perfkitbenchmarker/gcp/gce_virtual_machine.py

```diff
@@ -47,8 +48,8 @@
 FLAGS = flags.FLAGS
 
 SET_INTERRUPTS_SH = 'set-interrupts.sh'
-BOOT_DISK_SIZE_GB = 10
-BOOT_DISK_TYPE = disk.STANDARD
+BOOT_DISK_SIZE_GB = 50
+BOOT_DISK_TYPE = disk.REMOTE_SSD
```

Comment:
I can change it back to standard rather than ssd if you'd like - performance will be somewhat poor on Windows VMs in many cases if I do that. It does need a 50 GB boot disk because of the size of the image though.